.gitignore

```diff
@@ -1,2 +1,5 @@
 /vendor/
-composer.lock
\ No newline at end of file
+composer.lock
+.phpunit.result.cache
+/coverage/
+/reports/
```

Comment:
newline pls ^^

Reply:
i think we dont need a new line in .gitignore file